apps/retrospective/services/sprint_task.go

```diff
@@ -213,23 +213,46 @@ func (service SprintTaskService) tasksWithTimeDetailsForCurrentAndPrevSprint(ret
         users.first_name || ' ' || users.last_name          AS member_name,
         SUM(sprint_member_tasks.time_spent_minutes)
         OVER (
-          PARTITION BY tasks.id, sprint_members.member_id ) AS member_time`).
+          PARTITION BY tasks.id, sprint_members.member_id) AS member_time`).
 		Order("tasks.id").
-		Order("member_time DESC")
+		Order("member_time DESC").
+		Order("sprint_members.member_id DESC")
+
+	tempSprintTaskMemberTable := service.tasksForCurrentAndPrevSprint(retroID, sprintID).
+		Select(`
+        tasks.id as temp_task_id,
+		users.first_name || ' ' || users.last_name			AS sprint_member_name,
+		sprint_member_tasks.time_spent_minutes,
+		sprint_members.sprint_id,
+		sprint_members.member_id,
+        MAX(sprint_member_tasks.time_spent_minutes) OVER (PARTITION BY tasks.id, sprint_members.sprint_id) AS max_sprint_task_member_time,
+        SUM(sprint_member_tasks.time_spent_minutes) OVER (PARTITION BY tasks.id, sprint_members.member_id) AS sprint_task_member_total_time`)
+
+	sprintTaskOwnerTable := db.Raw(`
+        SELECT DISTINCT ON (temp_sprint_task_members.temp_task_id) temp_sprint_task_members.temp_task_id as task_id, *
```

Comment:
Refactor this as discussed for queries practices
Sample: https://github.com/iReflect/reflect-app/blob/4ab842ce7e0c1ed9807f08ff2c24ff9cca3a0ee3/apps/retrospective/services/sprint.go#L939